docs/sources/http_api/data_source.md

```diff
@@ -497,3 +497,110 @@ Content-Type: application/json
 `GET /api/datasources/proxy/:datasourceId/*`
 
 Proxies all calls to the actual data source.
+
+## Query a data source by id
+
+Queries a data source having backend implementation.
+
+`POST /api/tsdb/query`
+
+> **Note:** Most of Grafana's builtin data sources have backend implementation.
+
+**Example Request**:
+
+```http
+POST /api/tsdb/query HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+
+{
+  "from": "1420066800000",
+  "to": "1575845999999",
+  "queries": [
+    {
+      "refId": "A",
+      "intervalMs": 86400000,
+      "maxDataPoints": 1092,
+      "datasourceId": 86,
+      "rawSql": "SELECT 1 as valueOne, 2 as valueTwo",
+      "format": "table"
+    }
+  ]
+}
+```
+> **Note:** The `from`, `to`, and `queries` properties are required.
+
+JSON Body schema:
+
+- **from/to** – Should be either absolute in epoch timestamps in milliseconds or relative using Grafana time units. For example, `now-1h`.
+- **queries.refId** – Specifies an identifier of the query. Is optional and default to `"A".
```

Comment:
Either missing a closing single quote here or there is a single quote that should not be before the A.

Reply:
Fixed.